src/dsp_tools/commands/start_stack.py

```diff
@@ -45,6 +47,10 @@ def __post_init__(self) -> None:
             raise InputError(f"max_file_size must be between 1 and {MAX_FILE_SIZE}")
         if self.enforce_docker_system_prune and self.suppress_docker_system_prune:
             raise InputError('The arguments "--prune" and "--no-prune" are mutually exclusive')
+        if self.custom_host is not None and regex.match(
+            r"^(((\d{1,3}\.){3}\d{1,3})|(((([a-z]|\d|-|_|~)+\.)*([a-z]){2,})))$", self.custom_host
+        ):
```

Comment:
Now it does the opposite of what you intend. The regex should be negated. (It can also be slightly simplified, but that's not that important. The `not` is central):

```suggestion
        if self.custom_host is not None and not regex.match(
            r"^(((\d{1,3}\.){3}\d{1,3})|((([-\w_~]+\.)*([a-z]){2,})))$", self.custom_host
        ):
```

Reply:
BTW, have you tried out if this PR works as intended? You could install this on a server to try it out: `pip install git+https://github.com/rfbaumgartner/dsp-tools@param-for-ip`

Reply:
The `not` was missing all the time. I probably forgot to check the command after committing and after working on the regex I took a shortcut of only testing the expression itself.
Now, the tests seem to work as intended

Reply:
Perfect. I saw that you committed my suggestion. But then you overwrote them again during merging main.